
arch/m68k/q40/q40ints.c: In function 'q40_irq_handler':
arch/m68k/q40/q40ints.c:214: error: implicit declaration of function 'floppy_hardint'

FIXME q40_irq_handler() calls floppy_hardint():

arch/m68k/q40/q40ints.c:215: error: too few arguments to function 'floppy_hardint'

---
 arch/m68k/q40/q40ints.c |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/m68k/q40/q40ints.c
+++ b/arch/m68k/q40/q40ints.c
@@ -16,6 +16,7 @@
 #include <linux/errno.h>
 #include <linux/interrupt.h>
 
+#include <asm/floppy.h>
 #include <asm/ptrace.h>
 #include <asm/system.h>
 #include <asm/irq.h>
